### DISCUSSION TITLE
Contexts Managment

### DISCUSSION THREAD

**Discussion by felipe-mendieta** [infra-geo-ouverte/igo2#823] [Q&A]
Can I modify, add or perform actions on contexts from a graphical interface from igo2 lib?

Any control panel for the contexts? 

What is the difference between logging in as anonymous to igo2 and logging in with a username and password?

what is the username and password to test the auth module igo2-lib ?

Greetings.

**Comment by pelord** [comment]
I did not forget you. I will be ready soon to answer you correctly!
